src/MCPClient/lib/clientScripts/policy_check.py

```diff
@@ -313,14 +334,14 @@ def _get_command_to_execute(self, rule):
         else:
             return (rule.command.command, [self.file_path, self.policies_dir])
 
-    def _save_to_logs_dir(self, output):
+    def _save_to_logs_dir(self, output: Dict[str, str]) -> None:
```

Comment:
I think in the long term we need to parse `output = json.loads(stdout)` passing our own decoder that guarantees the types in the `output` mapping. See for example:

https://github.com/artefactual/archivematica/blob/01293dc7696ece0066353b6c83ce348ff4c3f3e5/src/MCPServer/lib/server/workflow.py#L228-L238